collect.py

```diff
@@ -197,7 +199,7 @@ def do_something(driver, elem_properties=None):
 
 def screenshot(driver, file_path):
     wait_loaded(driver)
-
+    file_path = "file:///home/anuja/tj/autowebcompat/test.html"
     driver.get_screenshot_as_file(file_path)
```

Comment:
You are changing where the screenshot must be saved, you need to change **driver.get** to load the URL (a few lines after).